### ISSUE TITLE
Release?

### ISSUE THREAD

**Issue by laryn** [backdrop-contrib/video_embed_field#3]
Is this module ready for a release so it shows up in the Project Installer?

**Comment by serundeputy** [comment]
@laryn in my opinion it is useable and useful, but i'm not willing to make a release w/ #1 and #2 known issues still outstanding.

**Comment by stpaultim** [comment]
I'm going to give it a try. I used this module in Drupal 7 a lot.

**Comment by olafgrabienski** [comment]
> it is useable and useful, but i'm not willing to make a release w/ #1 and #2 known issues still outstanding.

#2 is fixed, remains only one issue: #1

@laryn @stpaultim How is your experience with *Video Embed Field* so far?

**Comment by laryn** [comment]
@olafgrabienski My experience is that the defaults are working pretty well on the sites I am using this on, so the lack of a configuration page hasn't stopped me from using it.

**Comment by olafgrabienski** [comment]
Thanks for your feedback, @laryn. Do you also know by chance if the wysiwyg integration works? (Read about it on https://www.drupal.org/project/video_embed_field.)

**Comment by laryn** [comment]
@olafgrabienski I haven't used that so I don't know. I activated it on a dev site and enabled the text filter, but either I am not using it correctly or it's not working. I don't see an icon available in the WYSIWYG settings and haven't been successful in trying to add the code manually. Maybe open an issue for that to track and test.

**Comment by olafgrabienski** [comment]
@laryn Thanks a lot for looking into it! I'll try it also on a dev site and will post an issue if needed.

**Comment by olafgrabienski** [comment]
> My experience is that the defaults are working pretty well on the sites I am using this on, so the lack of a configuration page hasn't stopped me from using it.

In the meantime, I've tested Video Embed Field on a site I'm working on. The basics seem to work good, but https://github.com/backdrop-contrib/video_embed_field/issues/1 remains an issue for me because I'd like to use other video styles. Going to look for other approaches for now.

**Comment by laryn** [comment]
@olafgrabienski Did you end up testing the WYSIWYG integration?

**Comment by olafgrabienski** [comment]
@laryn Initially not, but now I gave the Wysiwig functionality a try, and it works. I've enabled the "Video Embedding" filter in the Full HTML text format (didn't see a respective button), and pasted examples with the following code patterns:

- `[VIDEO::https://vimeo.com/VIMEO-ID]`
- `[VIDEO::https://vimeo.com/VIMEO-ID::normal]`

Note: The separators have to consist of *two* colons (`::`).

YouTube worked as well. All videos are displayed quite small (440 x 225px), though, which is even smaller than the normal / fallback style when used as a field.
